src/test/kotlin/com/exactpro/th2/lwdataprovider/TestSseEventSearchRequest.kt

```diff
@@ -0,0 +1,93 @@
+package com.exactpro.th2.lwdataprovider
+
+import com.exactpro.cradle.TimeRelation
+import com.exactpro.th2.dataprovider.grpc.EventSearchRequest
+import com.exactpro.th2.lwdataprovider.entities.exceptions.InvalidRequestException
+import com.exactpro.th2.lwdataprovider.entities.requests.SseEventSearchRequest
+import com.google.protobuf.Timestamp
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class TestSseEventSearchRequest {
+    @Test
+    fun testConstructorParam(){
+        // startTimestamp - 1, endTimestamp - 2, searchDirection - AFTER (default)
+        var params = mutableMapOf("startTimestamp" to listOf("1"),
+            "endTimestamp" to listOf("2", "3"))
+        var eventSearchReq = SseEventSearchRequest(params)
+        Assertions.assertNotNull(eventSearchReq.startTimestamp)
+        Assertions.assertNotNull(eventSearchReq.endTimestamp)
+        Assertions.assertEquals(TimeRelation.AFTER, eventSearchReq.searchDirection)
```

Comment:
Please, extract each case in a separate test. The test must check only one condition. We should not combine several cases in one test.

Also, you can use nested tests in JUnit-jupiter. One for HTTP parameters and on for gRPC